### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by tyemacon**
My god it deploys right

### CODE REVIEW

File: Dockerrun.aws.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "Name": "tyemacon/carousel-proxy",
+    "Update": "true"
+  },
+  "Ports": [
+    {
+      "ContainerPort": "3000"
+    }
+  ],
+  "Logging": "/var/log/nginx"
+}
```
Comment (projectLewis) [inline Dockerrun.aws.json]: Whats this file do?

File: client/App.jsx [hunk 2]
```diff
@@ -1,16 +1,77 @@
...
+      test: 'Not Hello World',
+      url_75x75s: [],
+      url_170x135s: [],
+      url_570xNs: [],
+      url_fullxfull: [],
+      index: 0,
+    }
+  }
 
+  getImageUrls(){
```
Comment (projectLewis) [inline client/App.jsx]: consider putting these requests in a `componentDidMount() {}` in the future

File: client/App.jsx [hunk 3]
```diff
@@ -1,16 +1,77 @@
...
+    .catch((err) => {
+      if(err.response.status === 422){
+        console.log('No Product Found with that Id')
+      }else{
+        console.log('No images found for that product!')
+      }
+    })
   }
 
+  cycleImage(){
```
Comment (projectLewis) [inline client/App.jsx]: and this one in a `componentDidUpdate() {}`

componentDidUpdate takes prevState & prevProps, so you could tell it to run if the current `this.state.index` is not `prevState.index`

just considerations. I'm assuming this code is working, so I dont want you to break anything to follow convention.

File: Dockerrun.aws.json (hunk 1 above)
Comment (tyemacon) [reply Dockerrun.aws.json]: It creates the file path for any possible debugging/error logs. One of the explanations I was reading had it, figured it wouldn't hurt to include, though likely will be unnecessary.

**Pull request merged**